### DISCUSSION TITLE
Table grid lines not rendered with Markdown Editor

### DISCUSSION THREAD

**Discussion by juliuspopp** [requarks/wiki#3069] [Help / Questions] [answered]
**Describe the bug**

**To Reproduce**
1. Create a page with a table with Markdown Editor

**Expected behavior**
The table should have grid lines like a table created with the Visual Editor.

**Screenshots**
Markdown source and output:

![wiki-js-table-grid-not-rendered](https://user-images.githubusercontent.com/29792509/107325372-f0637800-6aa9-11eb-8efa-a3797f7b8431.png)

how I expect it to look and how tables from the visual editor render:

![grid-rendered-when-created-with-visual-editor](https://user-images.githubusercontent.com/29792509/107325382-f5c0c280-6aa9-11eb-8758-262069711718.png)

**Host Info (please complete the following information):**
 - OS: Debian 10
 - Wiki.js version: 2.5.170
 - Database engine: postgres 11.9

**Additional context**
- Rendering/MultiMarkdownTable enabled
 - node.js version: 12.20.1 
- Firefox 84.0.2

**Comment by regevbr** [comment, marked as answer]
@juliuspopp this is being controlled by the CSS of the theme. You can add your own custom CSS overrides in the theme section in the admin panel to control it. For example:
```css
.contents table td {
    border: 1px solid black;
}
.contents table th {
    border: 1px solid black;
}
```

**Reply by juliuspopp** [reply]
Ok thank you, that's a fix. I still find it weird (and would call it a bug) that the themes for the two editors are different though.
